src/frontend/qt_sdl/CMakeLists.txt

```diff
@@ -40,6 +40,7 @@ set(SOURCES_QT_SDL
     SaveManager.cpp
     CameraManager.cpp
     AudioInOut.cpp
+    ../../FrameDump.cpp
```

Comment:
Refresh my memory; what difficulty were you having compiling this file into the `core` target again?

Reply:
results in frontend files (specifically ROMManager.h) that were included in framedump.cpp being unable to find their includes
need a better solution to this problem